src/Interface/Modules/Visualization/CreateStandardColorMap.ui

```diff
@@ -233,6 +233,65 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="groupBox_2">
+     <property name="minimumSize">
+      <size>
+       <width>0</width>
+       <height>60</height>
+      </size>
+     </property>
+     <property name="title">
+      <string/>
+     </property>
+     <property name="alignment">
+      <set>Qt::AlignCenter</set>
+     </property>
+     <property name="flat">
+      <bool>false</bool>
+     </property>
+     <layout class="QHBoxLayout" name="horizontalLayout">
+      <property name="spacing">
+       <number>6</number>
+      </property>
+      <property name="leftMargin">
+       <number>2</number>
+      </property>
+      <property name="topMargin">
+       <number>2</number>
+      </property>
+      <property name="rightMargin">
+       <number>2</number>
+      </property>
+      <property name="bottomMargin">
+       <number>2</number>
+      </property>
+      <item>
+       <widget class="QPushButton" name="customColorButton0_">
```

Comment:
What do you mean by Custom Color 0 and Custom Color 1? One seems to be grayscale and the other is color? It doesn't seem straightforward to the user

Reply:
@allywarner Custom Color 0 is the color at the 0th element in the colormap and Custom Color 1 is the value of the 1st element in the colormap. Perhaps Custom Color Min and Custom Color Max could be more straight forward if that doesn't make sense to most people? Also, there should be no such limitation to either of them they use the exact same default color selectors provided by the target OS.